### ISSUE TITLE
Creative inventory crash

### ISSUE THREAD

**Issue by Craluminum2413** [anegostudios/VintageStory-Issues#1406]
**Game Version:** 1.15.8 
**Platform:** Windows
**Modded:** No
**SP/MP:** Singleplayer

### Description
Creative tabs are causing crash. I uninstalled mods and cleared cache but it still crashes. I updated the game without reinstalling

### How to reproduce
1. Join any world
2. Open creative menu
3. It crashes

### Logs
```
Version: v1.15.8 (Stable)11/07/2021 20:05:06: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.API.Client.GuiElementVerticalTabs..ctor(ICoreClientAPI capi, GuiTab[] tabs, CairoFont font, CairoFont selectedFont, ElementBounds bounds, Action`2 onTabClicked)
   at Vintagestory.API.Client.GuiComposerHelpers.AddVerticalTabs(GuiComposer composer, GuiTab[] tabs, ElementBounds bounds, Action`2 OnTabClicked, String key)
   at Vintagestory.Client.NoObf.GuiDialogInventory.ComposeCreativeInvDialog()
   at Vintagestory.Client.NoObf.GuiDialogInventory.ComposeGui(Boolean firstBuild)
   at Vintagestory.Client.NoObf.GuiDialogInventory.OnGuiOpened()
   at Vintagestory.API.Client.GuiDialog.TryOpen()
   at Vintagestory.API.Client.GuiDialog.OnKeyCombinationToggle(KeyCombination viaKeyComb)
   at _QJ4kUUHOLGfDzDzLIY07WLLlP0q._dbc1z1apNEld90b3JJZJCSTAxAI(KeyEvent , IWorldAccessor , IPlayer , Boolean , Boolean , Boolean )
   at Vintagestory.Client.NoObf.ClientMain.OnKeyDown(KeyEvent args)
   at Vintagestory.Client.NoObf.ClientPlatformWindows.game_KeyDown(Object sender, KeyboardKeyEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.Platform.NativeWindowBase.OnKeyDown(Key key, Boolean repeat)
   at OpenTK.Platform.Windows.WinGLNative.HandleKeyboard(IntPtr handle, WindowMessage message, IntPtr wParam, IntPtr lParam)
   at OpenTK.Platform.Windows.WinGLNative.WindowProcedure(IntPtr handle, WindowMessage message, IntPtr wParam, IntPtr lParam)
   at OpenTK.Platform.Windows.Functions.DispatchMessage(MSG& msg)
   at OpenTK.Platform.Windows.WinGLNative.ProcessEvents()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _KL43rdn8lsiOwB7iQoQbPYDj6tD._trfdlnkeEUMmEPQzhGYlxmsmaAt(_WBHLlCKhXzv2372dpFsWGEAhN0H , String[] )
   at _xfmEkYHtWw2EfeJcAohoAvJtA7O._trfdlnkeEUMmEPQzhGYlxmsmaAt(ThreadStart )
```

**Comment by Ahrondus** [comment]
Also having this issue after using the updater to update my game from 1.15.8-rc.1 to the stable version 1.15.8. Steps to reproduce are the same, crash log is the same (as you would imagine).

**Comment by Craluminum2413** [comment]
It turns out it doesn't matter if you updated or completely reinstalled the game. I reinstalled the game and it is still crashing

**Comment by Craluminum2413** [comment]
Okay it is fixed in 1.15.9
